src/Sarif/Schemata/sarif-schema.json

```diff
@@ -1947,6 +1993,17 @@
                     }
                 },
```

Comment:
>} [](start = 16, length = 1)

Per the mail I just forwarded to you: Please add a `string`-valued property `run.language` (at the location of this comment), described as "The language of the messages in the log file for this run". Thanks!

Reply:
hold on, is this a TC approved change? our current goal is to get a schema out that expresses the outcomes of the recent eballots.

Reply:
I see that it is not. It is fine to take the typo 'translation' to 'translations'. let's please hold off on the addition of extra properties. i very much would like a copy of the schema that snaps to the eballots.

Reply:
please note: i am not planning to add any post-ballot changes in these set of PRs, mostly to avoid massive confusion detangling which changes came from where. So, for now i am not going to make this change. On the post ballot items, i am working on a list of changes which both Mike and Larry have to agree on before i start taking up those.

---
In reply to: [263171079](https://github.com/Microsoft/sarif-sdk/pull/1336#discussion_r263171079) [](ancestors = 263171079)

Reply:
noted for post-ballot changes list.

---
In reply to: [263058987](https://github.com/Microsoft/sarif-sdk/pull/1336#discussion_r263058987) [](ancestors = 263058987)

Reply:
noting this in the post-ballot changes list.

---
In reply to: [263171718](https://github.com/Microsoft/sarif-sdk/pull/1336#discussion_r263171718) [](ancestors = 263171718)